### PR TITLE
More sdk work - Recipient/Folder

### PR DESCRIPTION
**Pull request by MartyIce**

### CODE REVIEW

**Comment by MartyIce** [comment]
[Iris.Sdk Migration](https://trello.com/c/9TW2Pjsu/548-irissdk-migration)

File: Iris.Sdk.Contracts/Recipient/GetRecipientAliasesResponse.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Iris.Sdk.Contracts.Recipient
+{
+    public class GetRecipientAliasesResponse : PagedResponse<RecipientAlias>
```
Comment (mogrodniczak) [inline Iris.Sdk.Contracts/Recipient/GetRecipientAliasesResponse.cs]: Do we need DataContract here?

File: Iris.Sdk.Contracts/Recipient/GetAddressRecipientsResponse.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+namespace Iris.Sdk.Contracts.Recipient
+{
+    public class GetAddressRecipientsResponse : PagedResponse<AddressRecipient>
```
Comment (mogrodniczak) [inline Iris.Sdk.Contracts/Recipient/GetAddressRecipientsResponse.cs]: Do we need DataContract here?

File: Iris.Sdk.Contracts/Recipient/GetRecipientAliasesResponse.cs (hunk 1 above)
Comment (MartyIce) [reply Iris.Sdk.Contracts/Recipient/GetRecipientAliasesResponse.cs]: I don't *believe* so, as the base class has it (I basically just copied code over).  However, probably good to have it explicit, so I've added to both

**Pull request merged**